control/freqplot.py

```diff
@@ -190,35 +190,15 @@ def bode_plot(syslist, omega=None,
     grid = config._get_param('bode', 'grid', kwargs, _bode_defaults, pop=True)
     plot = config._get_param('bode', 'grid', plot, True)
     margins = config._get_param('bode', 'margins', margins, False)
-    wrap_phase = config._get_param(
-        'bode', 'wrap_phase', kwargs, _bode_defaults, pop=True)
-    initial_phase = config._get_param(
-        'bode', 'initial_phase', kwargs, None, pop=True)
+    wrap_phase = config._get_param('bode', 'wrap_phase', kwargs, _bode_defaults, pop=True)
+    initial_phase = config._get_param('bode', 'initial_phase', kwargs, None, pop=True)
+    omega_num = config._get_param('freqplot', 'number_of_samples', omega_num)
```

Comment:
Please don't exceed 79 characters per line [(PEP8)](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

Reply:
Sorry, I just noticed that my IDE (PyCharm) uses 120 as default, which is not PEP!
Will change back :)